opencti-platform/opencti-front/src/private/components/common/stix_domain_objects/StixDomainObjectAttackPatternsKillChain.tsx

```diff
@@ -284,7 +284,7 @@ const StixDomainObjectAttackPatternsKillChain: FunctionComponent<StixDomainObjec
                 />
               </ToggleButton>
             </Tooltip>
-            <Tooltip title={t_i18n('Matrix in-line view')}>
+            <Tooltip title={t_i18n('Matrix in line view')}>
```

Comment:
To do : 
Remove 'Matrix in-line view' translations
Add 'Matrix in line view' translations